src/views/landing.njk

```diff
@@ -9,26 +8,21 @@
 {% endblock %}
 
 {% block beforeContent %}
-  {% block backLink %}{% endblock %}
-  <div class="govuk-breadcrumbs">
-    <ol class="govuk-breadcrumbs__list">
-      <li class="govuk-breadcrumbs__list-item">
-        <a class="govuk-breadcrumbs__link" href="https://www.gov.uk/">Home</a>
-      </li>
-      <li class="govuk-breadcrumbs__list-item">
-        <a class="govuk-breadcrumbs__link" href="https://www.gov.uk/browse/business">Business and self-employed</a>
-      </li>
-      <li class="govuk-breadcrumbs__list-item">
-        <a class="govuk-breadcrumbs__link" href="https://www.gov.uk/browse/business/limited-company">Running a limited company</a>
-      </li>
-    </ol>
-  </div>
+  {{
+    govukBreadcrumbs({
```

Comment:
I suggest we use macros as much as possible creating our own if needed. They nicely hide CSS complexity and make us focus on content, not mentioning reusability.